app/assets/javascripts/components/common/ArticleViewer/containers/ArticleViewer.jsx

```diff
@@ -189,13 +190,9 @@ const ArticleViewer = ({ showOnMount, users, showArticleFinder, showButtonLabel,
     if (editorsID.length) {
       // If there are unhighlighted editors, call the function to check their contributions in wikitext metadata
       usersContributionExists(editorsID);
-    } else {
-      const status = 'No Highlighted Editors';
-      // Set the unhighlightedEditors state with a status message to make legendStatus ready in the
-      // Footer Component for loading the authorship data
-      setUnhighlightedEditors([status]);
```

Comment:
We don't want to get rid of this 'unhighlighted editors' feature, as it's very useful to show the user that we successfully received data but none of the editor's work is part of the received data.

Reply:
Okay. If you see a good way to fix the bug such that the 'no highlghted editors' message shows appropriately (after data is successfully received but the relevant editors' IDs are not found in the data), please do so.